software/index.html

```diff
@@ -51,6 +51,7 @@ <h2>Version History</h2>
 </ul>
 
 <h2>Updating Runtime</h2>
+<i><p style="font-size:18px">Please only update Runtime if instructed by PiE Staff or if the Runtime section at the top of the page says to update.</p></i>
```

Comment:
I'm kinda meh on this mixing of HTML tags and inline CSS for styling, but I also realize I used inline CSS styling 3 lines down this year and this matches that.